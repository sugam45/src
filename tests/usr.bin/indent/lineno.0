/* $NetBSD: lineno.0,v 1.1 2021/03/08 20:12:04 rillig Exp $ */
/* $FreeBSD$ */

/* Demonstrates line number counting in verbose mode. */

int *
function(void)
{
}
