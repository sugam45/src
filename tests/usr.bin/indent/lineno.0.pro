/* $NetBSD: lineno.0.pro,v 1.1 2021/03/08 20:12:04 rillig Exp $ */
/* $FreeBSD$ */

-v
