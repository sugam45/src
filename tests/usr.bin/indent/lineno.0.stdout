/* $NetBSD: lineno.0.stdout,v 1.1 2021/03/08 20:12:04 rillig Exp $ */
/* $FreeBSD$ */

/* Demonstrates line number counting in verbose mode. */

int *
function(void)
{
}
/* $ In the below output, the '5' means 5 non-empty lines. */
/* $ XXX: It's rather strange that -v writes to stdout, */
/* $ even in filter mode.  This output belongs on stderr instead. */
There were 5 output lines and 1 comments
(Lines with comments)/(Lines with code):  0.250
